### PR TITLE
fixed mysql class/config name

### PR DESCRIPTION
**Pull request by nurdism**
fixed mysql class to accept config database name

### CODE REVIEW

File: src/main/java/net/mohron/skyclaims/database/MysqlDatabase.java [hunk 1]
```diff
@@ -40,8 +41,9 @@ public MysqlDatabase() {
 		databaseTablePrefix = config.getTablePrefix();
 		username = config.getUsername();
 		password = config.getPassword();
+        name = config.getDatabaseName();
 		port = config.getPort();
```
Comment (Mohron) [inline src/main/java/net/mohron/skyclaims/database/MysqlDatabase.java]: The code format is not consistent with the rest of the project.

File: src/main/java/net/mohron/skyclaims/database/Database.java [hunk 2]
```diff
@@ -35,24 +35,32 @@
...
+			String table =
+				"CREATE TABLE IF NOT EXISTS `" + this.tablename + "` ( " +
+				"`island` TEXT NOT NULL, "+
+				"`owner` TEXT NOT NULL, " +
+				"`claim` TEXT NOT NULL, " +
+				"`spawnX` INT NOT NULL, " +
+				"`spawnY` INT NOT NULL, " +
+				"`spawnZ` INT NOT NULL, " +
+				"`locked` BOOLEAN NOT NULL, " +
+				"PRIMARY KEY " + (type.equals("SQLite") ? "(`island`)" : "(`island`(60))") + ");";
```
Comment (Mohron) [inline src/main/java/net/mohron/skyclaims/database/Database.java:63]: Why the difference in primary keys between db types?

Comment (nurdism) [reply src/main/java/net/mohron/skyclaims/database/Database.java:63]: MySQL requires a length to a primary key that is type of TEXT and SQLite doesn't support adding length to a TEXT type.

Comment (Mohron) [reply src/main/java/net/mohron/skyclaims/database/Database.java:63]: I couldn't find documentation of that for some reason lol. Why 60 then? UUIDs are a fixed length, 36 with dashes, https://en.m.wikipedia.org/wiki/Universally_unique_identifier

Comment (nurdism) [reply src/main/java/net/mohron/skyclaims/database/Database.java:63]: because I was lazy and didn't want to look up how many where in a UUID :P

**Comment by Mohron** [comment]
Migration needs testing before this feature can be released.

**Pull request merged**